contracts/token/FTCVesting.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.6.10;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SafeMath } from "@openzeppelin/contracts/math/SafeMath.sol";
+
+/// @title A vesting contract for full time contributors
+/// @author 0xModene
+/// @notice You can use this contract to set up vesting for full time DAO contributors
+/// @dev All function calls are currently implemented without side effects
+contract FTCVesting {
+    using SafeMath for uint256;
+
+    address public index;
+    address public recipient;
+    address public treasury;
+
+    uint256 public vestingAmount;
+    uint256 public vestingBegin;
+    uint256 public vestingCliff;
+    uint256 public vestingEnd;
+
+    uint256 public lastUpdate;
+
+    constructor(
+        address index_,
+        address recipient_,
+        address treasury_,
+        uint256 vestingAmount_,
+        uint256 vestingBegin_,
+        uint256 vestingCliff_,
+        uint256 vestingEnd_
+    ) public {
+        require(vestingCliff_ >= vestingBegin_, "FTCVester.constructor: cliff is too early");
```

Comment:
What's the reasoning for removing this check in the Vesting contract?
`require(vestingBegin_ >= block.timestamp, "TreasuryVester.constructor: vesting begin too early");`

Is it retroactively vesting for contributors? Add a comment here noting why we removed the require for clarity.

Reply:
This was removed to allow retroactive vesting. We are currently approaching the cliff date on a few contributors, so this was removed to keep the accounting data straight with the hire date, vesting begin, etc.